src/scripts/i18n/write_updated_files.py

```
@@ -0,0 +1,23 @@
+import os
+import argparse
+
+if __name__ == '__main__':
+    parser = argparse.ArgumentParser(description="Write updated files for a given type.")
+    parser.add_argument('--type', type=str, default='components', choices=['components', 'services', 'pipes', 'directives'], help='Type of files (components, services, pipes, directives)')
+    args = parser.parse_args()
+
+    file_type = args.type
+    OUTPUT_JSON_PATH = f'src/scripts/i18n/{file_type}/output.json'
+
+    with open(OUTPUT_JSON_PATH, 'r', encoding='utf-8') as f:
```

Ruff (0.11.9)

Unnecessary mode argument

Remove mode argument

(UP015)